hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/RunCompactionActionExecutor.java

```diff
@@ -65,10 +73,14 @@ public RunCompactionActionExecutor(HoodieEngineContext context,
     this.operationType = operationType;
     checkArgument(operationType == WriteOperationType.COMPACT || operationType == WriteOperationType.LOG_COMPACT,
         "Only COMPACT and LOG_COMPACT is supported");
+    metrics = new HoodieMetrics(config);
   }
 
   @Override
   public HoodieWriteMetadata<HoodieData<WriteStatus>> execute() {
+    LOG.info("Compaction start.");
```

Comment:
do we need this. or you added for debugging purpose?

Reply:
@nsivabalan I added this based on our last discussion. It should help to get an idea of individual start / stop compaction events.